superbench/config/default.yaml

```diff
@@ -4,6 +4,10 @@ superbench:
   benchmarks:
     matmul:
       enable: true
+      modes:
+        - name: local
+          proc_num: 8
```

Comment:
What's the meaning of proc_num? It's not clear for user.

Reply:
process number